apps/web/__tests__/__mocks__/product.mock.ts

```diff
@@ -24,7 +24,12 @@ export const ProductMock = {
                 {
                     "url": "https://cdn02.plentymarkets.com/mevofvd5omld/item/images/157/full/gear-headphones-01.png",
                     "width": 1430,
-                    "names": [],
+                    "names": {
+                        "alternate": "Headphones Capybara",
+                        "imageId": 0,
+                        "lang": "de",
+                        "name": "Headphones Capybara",
+                    },
```

Comment:
```suggestion
                    "names": [],
```

I think this is no longer necessary.